custom_components/terncy/profiles/before_2023_7.py

```diff
@@ -6,7 +6,9 @@
 from homeassistant.helpers.entity import EntityCategory  # <2023.3
 
 from ..binary_sensor import TerncyBinarySensorDescription
+from ..climate import TerncyClimateDescription
 from ..const import (
+    PROFILE_10,
     PROFILE_11_LOCK,
     PROFILE_18,
```

Comment:
Suggest rename to PROFILE_OCCUPANCY_SENSOR

Reply:
Renamed PROFILE_18 to PROFILE_OCCUPANCY_SENSOR